helpers/github.ts

```diff
@@ -7,6 +7,10 @@ import { writeFile } from "fs/promises";
 import twitter from "./twitter";
 import { TwitterMap } from "..";
 
+export const DEVPOOL_OWNER_NAME = process.env.DEVPOOL_OWNER_NAME!;
+export const DEVPOOL_REPO_NAME = process.env.DEVPOOL_REPO_NAME!;
+export const IS_RFC = JSON.parse(process.env.RFC!);
```

Comment:
Non-null assertion operator is considered a bad practice because if the value is `null` then the app fails while the compiler is trying to make sure that even if the value is `null` the app works as expected (or at least throws meaningful errors). The `value!` syntax is not used anywhere in the codebase. Pls refactor accordingly.

Reply:
I imagine that the linter settings are not in sync with our template here, given that this passed CI

Reply:
> I imagine that the linter settings are not in sync with our template here, given that this passed CI.
> 
> Perhaps this is technically our responsibility to handle these sort of problems.

Seems you've already created a related issue https://github.com/ubiquity/ts-template/issues/54